public/app/features/dashboard-scene/utils/urlBuilders.ts

```diff
@@ -54,6 +56,10 @@ export function getDashboardUrl(options: DashboardUrlOptions) {
     };
   }
 
+  if (options.isHomeDashboard) {
```

Comment:
Why do we build the paths like this instead of using `location.pathname`? I must be missing something here

Reply:
@mdvictor what do you mean? this is used for many different scenarios (like generating URLs to solo panels which has different pathname)

Also a new dashboard has no uid so also has a unique pathname. What would your alternative be?

Reply:
Indeed, for the scenarios where we need to build the url it makes sense, like the share tab. But for dashboardScene [here](https://github.com/grafana/grafana/blob/13a81ea929d4fe719fdc1502430a1c5ccaf309c1/public/app/features/dashboard-scene/scene/DashboardScene.tsx#L394) couldn't we just pass the pathname and build the query params on top of that?

Why are we deducing the path based on `uid` and `meta` info instead of just getting the base `location.pathname` and adding query params?